digital_image_processing/morphological_operations/dilation_operation.py

```diff
@@ -61,14 +61,12 @@ def dilation(image: np.array, kernel: np.array) -> np.array:
     return output
 
 
-# kernel to be applied
-structuring_element = np.array([[0, 1, 0], [1, 1, 1], [0, 1, 0]])
-
-
 if __name__ == "__main__":
     # read original image
-    image = np.array(Image.open(r"..\image_data\lena.jpg"))
-    output = dilation(gray2binary(rgb2gray(image)), structuring_element)
+    lena = np.array(Image.open(r"..\image_data\lena.jpg"))

```

Comment:
Please make this a `pathlib.Path` instead of a Windows-specific raw string.
* https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open

Please run `ruff --select=PTH digital_image_processing/morphological_operations/dilation_operation.py` and make sure that it passes.